addons/web/static/src/views/graph/graph_arch_parser.js

```diff
@@ -22,6 +22,9 @@ export class GraphArchParser extends XMLParser {
                     if (node.hasAttribute("stacked")) {
                         archInfo.stacked = archParseBoolean(node.getAttribute("stacked"));
                     }
+                    if (node.hasAttribute("cumulated")) {
+                        archInfo.cumulated = archParseBoolean(node.getAttribute("cumulated"));
+                    }
```

Comment:
You could replace cumulated: "cumulated" in archInfo ? archInfo.cumulated : false,
By archInfo.cumulated = node.getAttribute("cumulated") ? archParseBoolean(node.getAttribute("cumulated")) : false